sdk/connect/javascript-wagmi.md

```diff
@@ -1,9 +1,10 @@
 ---
 description: Quickstart guide for using the SDK with a JavaScript and Wagmi dapp.
 toc_max_heading_level: 2
+sidebar_label: JavaScript + Wagmi (recommended)
```

Comment:
Any specific reason to recommend `wagmi` and not our own MM SDK?

Reply:
If you look at the guide, this uses the MetaMask SDK default template which spins up a Next.js and Wagmi dapp. This label is meant to identify the platform used with the SDK. We can consult with the team at our next docs sync if a different label would be more clear.